### DISCUSSION TITLE
XML Documentation for Generated Source

### DISCUSSION THREAD

**Discussion by josephdrake-stahls** [CommunityToolkit/dotnet#180] [Q&A] [answered]
Our organization's coding standards rely on XML Documentation for public members.  As of right now, the generated properties for fields decorated with `[ObservableProperty]` doesn't forward documentation from the field (and I'm not sure if its even possible).

Does anyone know of a way to document these properties inline?  The only potential solution I've found is to use an `<include />` tag to reference external xml documentation, but its not very maintainable in event of changes/refactoring.

**Comment by Sergio0694** [comment, marked as answer]
Hey there! This has been fixed by https://github.com/CommunityToolkit/dotnet/pull/124.
It will be available in 8.0.0-Preview3 🙂

**Reply by josephdrake-stahls** [reply]
Much appreciated!
